src/ocean/util/app/DaemonApp.d

```diff
@@ -652,6 +666,18 @@ public abstract class DaemonApp : Application,
         this.stats_ext.stats_log.add(this.system_stats.collect());
     }
 
+    /***************************************************************************
+
+        Collects GC stats and reports them to stats log. Should be
+        called periodically (inside onStatsTimer).
+
+    ***************************************************************************/
+
+    protected void reportGCStats ( )
```

Comment:
This new method should be mentioned in the release notes. (Didn't you have this before?)

Reply:
I think I lost the change during a rebase... I added it back